data/json/recipes/recipe_medsandchemicals.json

```
@@ -894,6 +894,21 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "platinum_small", -1 ] ] ],
     "components": [ [ [ "water_clean", 1 ] ], [ [ "chem_sulphur", 375 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "chem_sulphuric_acid",
+    "id_suffix": "with blood",
```

```suggestion
    "id_suffix": "with_blood",
```
Spaces in suffixes are valid, but also bad form.